paf-mvp-core-js/src/model/model.ts

```diff
@@ -20,3 +21,8 @@ export interface PAFNode {
   hostName: string;
   privateKey: string;
 }
+
+export interface RedirectErrorResponse {
+  code: ResponseCode;
+  error: NodeError; // FIXME[error] should be Error. JSON schema of Error to be updated to match NodeError
```

Comment:
From this FIXME, your intent isn't clear to me. What do you expect from the JSON Schema and errors? From what I see (and what should be), the errors of the JSON Schema Service have dedicated errors that are unboxed to create a `NodeError` within the handler (in `checkJsonBody`).

Reply:
Ok. Thank you for the clarification